rulesets/src/collections.ruleset.yml

```diff
@@ -64,15 +66,16 @@ rules:
                     in:
                       const: query
 
-  sps-post-request-body-missing-paging-object:
-    description: "POST collection endpoints MUST have a request body schema that includes paging parameters."
-    severity: error
-    given: $.paths[?(!@property.match(/.*\/\{[^}]+\}$/))].post.requestBody.content.application/json.schema.properties.paging
-    then:
-      field: "type"
-      function: pattern
-      functionOptions:
-        match: "object"
+  # sps-post-request-body-missing-paging-object:
+  # See https://atlassian.spscommerce.com/browse/DPE-286
+  #   description: "POST collection endpoints MUST have a request body schema that includes paging parameters."
+  #   severity: error
+  #   given: $.paths[?(!@property.match(/.*\/\{[^}]+\}$/))].post.requestBody.content.application/json.schema.properties.paging
```

Comment:
this def feels like we can keep - if you have something named paging, it better be an object.

Reply:
Re-enabled!